templates/security/cve/_cve-table.html

```diff
@@ -1,83 +1,52 @@
-                <span class="capitalize">{{ cve.priority }}</span>
-              </div>
-              <div class="icon-container__icon icon-cve-info">
-                <i class="p-icon--information cve-tooltip-icon cve-tooltip-icon-priority" data-priority="{{ cve.priority }}"></i>
-              </div>
+    {% for package in cve.packages %}
+      <tr class={% if loop.index > 5 %}"expandable-row u-hide"{% endif %}>
+        <td>{{ package.name }}</td>
+        {% for release in selected_releases|reverse %}
+          {% if versions|length == 0 %}
+            {% if release.codename == "noble" or
+                  release.codename ==  "mantic" or
+                  release.codename == "jammy" or
+                  release.codename == "focal" or
+                  release.codename == "bionic" %}
+              {% set status = package.release_statuses[release.codename] %}
+            {% endif %}
+          {% else %}
+            {% set status = package.release_statuses[release.codename] %}
+          {% endif %}
+          {% if status %}
+            <td class={% if status.slug == "DNE" %}"u-text--muted"{% endif %}>
+              <i class="p-icon--{{ status.icon }}"></i> {{ status.name }}
             </td>
+          {% elif versions|length > 0 or (versions|length == 0 and loop.index0 < 5) %}
+            <td>&mdash;</td>
           {% endif %}
-          <td>{{ package.name }}</td>
-          {% for release in selected_releases %}
-            {% set status = package.release_statuses[release.codename] %}
-            {% if status %}
-            
-              <td class="cve-table-cell status-{{ status.slug }}">
-                <div class="cve-color-strip--{{ status.slug.lower() }}"></div>
-                <div class="icon-container__text">
-                  {{ status.name}}
-                  {% if status.pocket == "esm-infra" %}
-                    <small class="cve-table-cell esm-info"><a href="/pro">Ubuntu Pro/Ubuntu Pro (Infra-only)</a></small>
-                  {% endif %}
-                  {% if status.pocket == "esm-apps" %}
-                    <small class="cve-table-cell esm-info"><a href="/pro">Ubuntu Pro</a></small>
-                  {% endif %}
-                </div>
-                <div class="icon-container__icon icon-cve-info">
-                  {% if status.status == 'pending' or status.slug == "not-affected" %}
-                    <i class="p-icon--information cve-tooltip-icon cve-tooltip-status" data-status="{{ status.slug }}"></i>
-                  {% endif %}
-                </div>
-              </td>
-            {% else %}
-              <td class="cve-table-cell--muted">
-                <div class="cve-color-strip--dne"></div>
-                &mdash;
-              </td>
-            {% endif %}
-          {% endfor %}
-        </tr>
-      {% endfor %}
+        {% endfor %}
+      </tr>
     {% endfor %}
+    <tr />
   </tbody>
 </table>
+{% if cve.packages|length > 5 %}
```

Comment:
Can you also add the slight fade at the bottom as in the design? From the looks of it it should be added when there are more than 6 packages in the table but I'm not100% sure if that's accurate, @juanruitina could you confirm?
![image](https://github.com/canonical/ubuntu.com/assets/17607612/f0b09338-0581-46ff-b2ab-f873eead2ede)

Reply:
Yes! That's right, that fading should disappear of course when clicking on "Show all x packages"